components/statuslight/index.css

```diff
@@ -22,11 +24,12 @@ governing permissions and limitations under the License.
   flex-direction: row;
   align-items: flex-start;
 
-  padding: var(--spectrum-statuslight-padding-y) 0px;
+  padding: calc(var(--spectrum-statuslight-padding-y) - 1px) 0 calc(var(--spectrum-statuslight-padding-y) + 1px) 0;
```

Comment:
Maybe move to root like we did here and add a little comment:
https://github.com/adobe/spectrum-css/pull/412/files#diff-b0e8fec1859b9f098bf3ffc22f003453R16